server/channels/utils/textgeneration.go

```diff
@@ -468,11 +470,23 @@ Up to and hey without pill that this squid alas brusque on inventoried and sprea
 func RandString(l int, charset string) string {
 	ret := make([]byte, l)
 	for i := 0; i < l; i++ {
-		ret[i] = charset[rand.Intn(len(charset))]
+		ret[i] = charset[mrand.Intn(len(charset))]
 	}
 	return string(ret)
 }
 
+func SecureRandString(n int) string {
+	charset := "abcdedfghijklmnopqrstABCDEFGHIJKLMNOPQRSTUVWXYZ!@#$%&*0123456789"
```

Comment:
`d` appears twice in the charset which can create a bias: 'd' will appear approximately twice as often as other characters.

```suggestion
	charset := "abcdefghijklmnopqrstABCDEFGHIJKLMNOPQRSTUVWXYZ!@#$%&*0123456789"
```

Reply:
Good catch! Taking a second look at the charset, I saw that we were missing "uvwxyz" on the lower case letters, added them at [b8d09cc](https://github.com/mattermost/mattermost/pull/27864/commits/b8d09cc5f00fa2f7b5f8eb1f1e776cadfe1b8cef), PTAL